### DISCUSSION TITLE
Troubleshooting on AWS

### DISCUSSION THREAD

**Discussion by schbz** [plausible/analytics#2086] [Self-Hosted Support] [answered]
Hi, I've got everything setup and apparently running on an Ubuntu AWS instance but  there doesn't seem to be anything being served on the IP and port specified. I've double checked all the ENV variables and the firewalls and everything. The output on the terminal looks like it might be stuck on something because it stops after the following lines:

`plausible_events_db_1  | 2022.08.10 00:05:04.587998 [ 44 ] {} <Warning> Application: Listen [::]:9004 failed: Poco::Exception. Code: 1000, e.code() = 0, e.displayText() = DNS error: EAI: Address family for hostname not supported (version 21.3.2.5 (official build)). If it is an IPv6 or IPv4 address and your host has disabled IPv6 or IPv4, then consider to specify not disabled IPv4 or IPv6 address to listen in <listen_host> element of configuration file. Example for disabled IPv6: <listen_host>0.0.0.0</listen_host> . Example for disabled IPv4: <listen_host>::</listen_host>plausible_1            | Loading plausible..plausible_1            | Starting dependencies..plausible_1            | Starting repos..plausible_1            | create Plausible.Repo database if it doesn't existplausible_1            | create Plausible.ClickhouseRepo database if it doesn't existplausible_1            | Creation of Db successful!plausible_1            | Loading plausible..plausible_1            | Starting dependencies..plausible_1            | Starting repos..plausible_1            | Running migrations for Elixir.Plausible.Repoplausible_1            | Running migrations for Elixir.Plausible.ClickhouseRepoplausible_1            | Migrations successful!plausible_1            | Loading plausible..plausible_1            | Starting dependencies..plausible_1            | Starting repos..plausible_1            | Admin user already exists. I won't override, bailing`

Then nothing... Any help with this would be much appreciated.

**Comment by schbz** [comment, marked as answer]
Nevermind! There was a typo is all. It's working beautifully with AWS lightsail and Cloudfront

**Comment by mir1198yusuf** [comment]
@schbz is mailer working on ec2 ? i just used default smtp config, no relay, but getting this error 
`Connecting to aspmx.l.google.com [2404:6800:4003:c1c::1b]:25 ... failed: Cannot assign requested address`

**Reply by ruslandoga** [reply]
👋 @mir1198yusuf 

Sorry for a late reply, but this is probably due to AWS EC2 blocking outgoing connections to port 25. You would need to open a ticket with AWS support to allow it.
